### PR TITLE
Create PythonToHash

### PR DESCRIPTION
**Pull request by Anon007-cyber**

### CODE REVIEW

File: PythonToHash [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Python rogram to find the SHA-1 message digest of a file
```
Comment (kashifkhan0771) [inline PythonToHash:1]: Python program to find the SHA-1 message digest of a file( you missed a word `p` in program
And keep this comment below imports for better readability

**Review by kashifkhan0771 — APPROVED** [review]

File: PythonToHash [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+# importing the hashlib module
+import hashlib
```
Comment (bupboi1337) [inline PythonToHash:4]: If your importing modules always remember to keep a requirments.txt listing the modules imported in the same folder as the program itself.

File: PythonToHash [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+       chunk = 0
+       while chunk != b'':
+           # read only 1024 bytes at a time
+           chunk = file.read(1024)
+           h.update(chunk)
+
+   # return the hex representation of digest
+   return h.hexdigest()
+
+message = hash_file("track1.mp3")
```
Comment (bupboi1337) [inline PythonToHash:26]: could replace message with just m and comment that m means message for smaller filesize.

File: PythonToHash (hunk 1 above)
Comment (bupboi1337) [inline PythonToHash:1]: typo but good description of the program overall with this single comment.

**Review by bupboi1337 — CHANGES_REQUESTED** [review]
### Its good but needs some changes
This looks like a wonderful script but it does have some issues that can be fixed.
All it needs is the following for it to check out in my book are the following things:

- The program to be in its own folder
-  A requirments.txt listing the module(s) used (in this case it would be the hashlib module)
- And last but not least some minor optimization and rewording

After all that is added this would be a great addition to add to this repo.
**tl;dr it looks great but needs a bit more work**

**Review by NitkarshChourasia — APPROVED** [review]
Properly documented.
Properly named.
Done, following the best practices and standards of python programming language, indeed.
